apps/network-alerts/test/unit/attention/attention_test.js

```diff
@@ -103,11 +100,14 @@ suite('Network Alerts - Attention Screen', function() {
     });
 
     test('Notification should be displayed and alert is played', function() {
+      this.sinon.stub(NotificationHelper, 'send');
```

Comment:
If we use `stub` here, we'll need to return a fake notification in resolve:
```
this.sinon.stub(NotificationHelper, 'send').returns(Promise.resolve({}));
```
Ideally we should verify the onerror/onshow callback as well, but it'll need to refactor attention init for returning the setNotification promise.